src/app/containers/Metadata/index.jsx

```diff
@@ -160,72 +101,60 @@ const MetadataContainer = ({ metadata, promo }) => {
     icon: ['72x72', '96x96', '192x192'],
   };
 
-  const title =
-    pageType === 'frontPage' && frontPageTitle
-      ? frontPageTitle
-      : getTitle(promo);
-
   return (
     <>
       <LinkedData
```

Comment:
Since `<LinkedData>` is rendered in the Metadata container, we pass down `pageSpecificLinkedData` down through four different files for articles. 

```
containers/ArticleMain 
components/ArticleMetadata
containers/Metadata
components/LinkedData
```

I'd suggest pulling out LinkedData from the Metadata container and just using it at the page level where it is needed. 

So, `containers/ArticleMain ` will import the `LinkedData` component directly. This will simplify the logic so we don't need to go through so many files for rendering LinkedData.

Reply:
I like this idea. This means there is a disconnect between metadata and linked data though. I have a pull request that renders the linked data directly in the metadata container.

https://github.com/bbc/simorgh/pull/4094/files

Page types that don't have page specific linked data don't need to worry about linked data as it will be included with the metadata container. Currently it's only articles that have specific linked data.

What do you think is best?

Reply:
I think having LinkedData treated separately to Metadata shouldn't be a problem - it'd make it simpler for developers working on each page type to see what's being included, since in the `containers/ArticleMain` you'll have LinkedData as well as Metadata, but in `containers/FrontPageMain` you just have Metadata.

Reply:
@sareh just to clarify, all page types actually have linked data. It's just articles have some extra linked data. Was that your understanding in the last comment?

Reply:
@jroebu14 Yes, that's right.